_example/lambda.go

```diff
@@ -15,20 +16,20 @@ type MyEvent struct {
 	Name string `json:"name"`
 }
 
-func HandleRequest(ctx context.Context, name MyEvent) (string, error) {
+func HandleRequest(ctx context.Context, name MyEvent) (events.APIGatewayProxyResponse, error) {
 	response := fmt.Sprintf("Hello %s!", name.Name)
 	returnErr, ok := os.LookupEnv("RETURN_ERROR")
 	if !ok {
-		return response, nil
+		return events.APIGatewayProxyResponse{Body: response, StatusCode: 200}, nil
 	}
 	isReturnErr, err := strconv.ParseBool(returnErr)
 	if err != nil {
-		return response, nil
+		return events.APIGatewayProxyResponse{Body: response, StatusCode: 500}, err
 	}
 	if isReturnErr {
-		return "", errors.New("failed error")
+		return events.APIGatewayProxyResponse{Body: response, StatusCode: 500}, errors.New("failed error")
```

Comment:
How does it look in the transaction page? Does it look ok?
We mark 500 return-value as an error in the graph, here we have both 500 and `error`.

Reply:
Shows as an error. it doesn't matter right? it's a business logic error where you return 500 but in parallel a business error.